credits.html

```diff
@@ -0,0 +1,274 @@
+    Astropy around 2015. Since 2017 he has been the main maintainer of io.fits,
+    taking on the daunting role of managing this complex and critical
+    subpackage. He also contributes other parts of Astropy including
+    infrastructure, modeling, io.ascii, stats, table, and visualization.
+
+    <h3>Tom Aldcroft</h3>
+    In 2011, Tom was part of an initial core group that recognized the need for
+    a common Astropy package for the community, and he helped organize the first
+    official Astropy coordination meeting. Since that time he has been an active
+    contributor to the project, taking a lead role in the development and
+    maintenance of three core subpackages: table, time, and io.ascii. In 2016 he
+    was appointed as one of the Astropy Project Coordination Committee members.
+
+    <h3>Tom Robitaille</h3>
+    Tom has been a recognized leader in the Astropy project since it began,
+    being part of the core group that started the project and organized the
+    first Astropy coordination meeting. His individual contributions are too
+    numerous to name, but they include contributing large parts of the core
+    package covering many areas, developing the astropy-healpix and regions
+    coordinated packages, serving as a release manager, GSoC coordinator, and
+    member of the Coordination Committee.
+
+<section>
+
+<h1 id="institutional-support">Institutional Support<a class="paralink"
+href="#institutional-support" title="Permalink to this headline">¶</a></h1>
+
+Here we recognize the institutions who have made major contributions to the
+Astropy project by either direct funding to the project or by indirect funding
+of employees who have contributed.
+
+<h3>Space Telescope Science Institute (STScI)</h3>
+STScI has played a foundational role in the development and advancement of the
+Astropy Project since its inception. STScI has provided continued and
+substantial support to the project via staff contributions since 2011, including
+X lifetime contributors to astropy.  Additionally, STScI has provided on-going
+support and leadership to both Astropy and the  broader scientific Python
+computing ecosystem.
+
+<h3>Moore Foundation</h3>
+In late 2019 the Astropy project was awarded a major grant from the Gordon and
+Betty Moore Foundation. This grant was targeted at supporting Astropy’s
+transition to a fully sustainable project, where success no longer hinges on a
+limited set of contributors. This grant was transformative for the Project.
+
+<h3>Chandra X-ray Center (CXC)</h3>
+The CXC has supported multiple staff members to work on the Astropy project,
+equivalent to   more than 5 person-years since the start of the project.
+
+<h3>NumFocus</h3>
+We wish to express gratitude to the NumFocus Organization for providing the
+organizational support to grow Astropy into the role of a community-leading
+project with a substantial budget.
+
+<h3>Center for Computational Astronomy (CCA), Flatiron Institute, Simons
```

Comment:
Is this big enough to be mentioned? If so, I think the $30k grant from the Dunlap Institute should be mentioned too.

Though my sense would be to have a cut-off at ~1 person year.

Reply:
And then DiRAC/eScience, too. And the Lorentz Centre probably, too.

Reply:
Agreed with @mhvk and @bsipocz! Also minor tweak:
```suggestion
<h3>Center for Computational Astrophysics (CCA), Flatiron Institute, Simons
```